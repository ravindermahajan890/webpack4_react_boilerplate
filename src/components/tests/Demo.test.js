import React from "react";
import renderer from "react-test-renderer";
import Home from "../Demo";

it("renders correctly", () => {
  const tree = renderer.create(<Home name="Ravinder" />).toJSON();
  expect(tree).toMatchSnapshot();
});
