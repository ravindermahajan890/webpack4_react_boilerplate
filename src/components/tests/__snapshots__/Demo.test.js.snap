// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="hell"
>
  <div>
    Welcome 
    Ravinder
  </div>
  <img
    src="../../images/bitmap.png"
  />
</div>
`;
